jedi.el

```diff
@@ -83,13 +83,12 @@ in their Emacs configuration."
 ;; Calling `auto-complete' or `ac-update-greedy' instead of `ac-start'
 ;; here did not work.
 
-(defun jedi:dot-complete ()
+(defun jedi:dot-complete (prefix-arg)
```

Comment:
Would you use change argument name instead of `prefix-arg` ?
`prefix-arg` causes byte-compile warnings.

```
jedi.el:86:1:Warning: Lexical argument shadows the dynamic variable prefix-arg
```